app/App.js

```diff
@@ -383,7 +371,7 @@ export default class App extends Component {
 
     return !loading && !isHide && (
       <div
-        className={`container ${XcooBee.kit.config.position || "left_bottom"} ${!isShown ? "transparent" : ""}`}
+        className={`xb-cookie-kit__container ${XcooBee.kit.config.position} ${!isShown ? "transparent" : ""}`}
```

Comment:
Let's change `xb-cookie-kit__container` to `xb-cookie-kit` since this `div` and its children represent the XCK component.
```suggestion
        className={`xb-cookie-kit ${XcooBee.kit.config.position} ${!isShown ? "transparent" : ""}`}
```
Alternatively, we can just add `xb-cookie-kit`
```suggestion
        className={`xb-cookie-kit xb-cookie-kit__container ${XcooBee.kit.config.position} ${!isShown ? "transparent" : ""}`}
```

We'll need to update the class in `CookieKit.js` too.